apps/examples/testcase/le_tc/network/tc_net_ioctl.c

```diff
@@ -33,7 +33,7 @@
 
 #include "tc_internal.h"
 
-extern int ioctlsocket(int s, long cmd, void *argp);
+extern int ioctl(int s, long cmd, void *argp);
```

Comment:
Let's include appropritary header instead of extern.

Reply:
ok